.gitmodules

```diff
@@ -2,8 +2,8 @@
   path = spack
   #url = https://github.com/spack/spack
   #branch = develop
-  url = https://github.com/jcsda/spack
-  branch = jcsda_emc_spack_stack
+  url = https://github.com/AlexanderRichert-NOAA/spack
```

Comment:
The spack PR has been merged, please revert .gitmodules and update the submodule pointer. Thanks!